### PR TITLE
Feature/crd 634 26 add logo

### PR DESCRIPTION
**Pull request by joshfedo**

### CODE REVIEW

File: Block/PopulateFpx.php [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    ) {
+        parent::__construct($context, $data);
+        $this->assetRepository = $assetRepository;
+        $this->request = $request;
+    }
+
+
+
+
+    public function getFpxConfig()
```
Comment (chris-huffman) [inline Block/PopulateFpx.php]: @joshfedo  - can you add some docblocks to this file and clean up the spacing?

File: etc/adminhtml/system.xml [hunk 2]
```diff
@@ -3,7 +3,9 @@
     <system>
         <section id="payment">
             <group id="credova" translate="label" type="text" sortOrder="50" showInDefault="1" showInWebsite="1" showInStore="1">
-                <label>Credova</label>
+                <label>Credova Financing</label>
+                <fieldset_css>complex credova-section</fieldset_css>
+                <frontend_model>Magento\Paypal\Block\Adminhtml\System\Config\Fieldset\Payment</frontend_model>
```
Comment (chris-huffman) [inline etc/adminhtml/system.xml]: @joshfedo  - is there are reason we're depending on a PayPal class?

File: Block/PopulateFpx.php (hunk 1 above)
Comment (joshfedo) [reply Block/PopulateFpx.php]: Taken care of!

File: etc/adminhtml/system.xml (hunk 2 above)
Comment (joshfedo) [reply etc/adminhtml/system.xml]: That's the block that adds the configure button for the image. I've copied forward the file into credova and everything is looking good now.

File: Block/PopulateFpx.php [hunk 3]
```diff
@@ -9,6 +9,7 @@
 
 
 use Magento\Framework\View\Asset\Repository as AssetRepository;
+use Magento\Tests\NamingConvention\true\string;
```
Comment (chris-huffman) [inline Block/PopulateFpx.php]: @joshfedo  - can you remove this? I'm pretty sure you're not using this.

File: Block/Adminhtml/System/Config/Form/Fieldset/Payment.php [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+
+    $groupConfig = $element->getGroup();
+
+    $disabledAttributeString = $this->_isPaymentEnabled($element) ? '' : ' disabled="disabled"';
+    $disabledClassString = $this->_isPaymentEnabled($element) ? '' : ' disabled';
+    $htmlId = $element->getHtmlId();
+    $html .= '<div class="button-container"><button type="button"' .
+        $disabledAttributeString .
+        ' class="button action-configure' .
+        (empty($groupConfig['paypal_ec_separate']) ? '' : ' paypal-ec-separate') .
```
Comment (chris-huffman) [inline Block/Adminhtml/System/Config/Form/Fieldset/Payment.php]: @joshfedo  - there's a few paypal references left in here. Are those going to cause problems?

File: Block/Adminhtml/System/Config/Form/Fieldset/Payment.php [hunk 5]
```diff
@@ -0,0 +1,181 @@
...
+        $disabledClassString .
+        '" id="' .
+        $htmlId .
+        '-head" onclick="paypalToggleSolution.call(this, \'' .
```
Comment (chris-huffman) [inline Block/Adminhtml/System/Config/Form/Fieldset/Payment.php]: @joshfedo - does this function call to the method created below? That you seemed to rename for credova?

File: Block/PopulateFpx.php (hunk 3 above)
Comment (joshfedo) [reply Block/PopulateFpx.php]: Removed

File: Block/Adminhtml/System/Config/Form/Fieldset/Payment.php (hunk 4 above)
Comment (joshfedo) [reply Block/Adminhtml/System/Config/Form/Fieldset/Payment.php]: Looks like that id wasn't be used in this case but I'm going to change it to be safe.

File: Block/Adminhtml/System/Config/Form/Fieldset/Payment.php (hunk 5 above)
Comment (joshfedo) [reply Block/Adminhtml/System/Config/Form/Fieldset/Payment.php]: This is responsible for making the configure button show/hide the module settings. Ive changed the onclick and also where paypalToggleSolution is defined.

**Pull request merged**